CraftStore.lua

```diff
@@ -3170,7 +3170,7 @@ function CS.GetTrait(link)
   elseif wt==WEAPONTYPE_HEALING_STAFF then
     craft=6;line=5;
   elseif wt==WEAPONTYPE_SHIELD then
-    craft=6;line=6;trait=trait-10;
+    craft=6;line=6;
```

Comment:
Ah, This change is fixing nirnhoned, but breaks other trait like divines and so on..